test/e2e/tests/checkout.spec.ts

```diff
@@ -66,22 +72,23 @@ test.describe(
         paddleFrame = new PaddleFrame(page);
 
         // confirm that modal opens
-        expect(paddleFrame.modal).toContainText('Order summary');
+        await expect(paddleFrame.modal).toContainText('Order summary');
 
         // fill out form
-        await paddleFrame.emailField.fill(CHECKOUT_EMAIL_ADDRESS, { timeout: TIMEOUT_2_SECONDS });
+        await paddleFrame.emailField.fill(CHECKOUT_EMAIL_ADDRESS, { timeout: TIMEOUT_30_SECONDS });
         await paddleFrame.countryField.selectOption(CHECKOUT_COUNTRY);
-        await paddleFrame.postalCodeField.fill(CHECKOUT_POSTAL_CODE, { timeout: TIMEOUT_2_SECONDS });
+        await paddleFrame.postalCodeField.fill(CHECKOUT_POSTAL_CODE, { timeout: TIMEOUT_5_SECONDS });
         await paddleFrame.continueButton.click();
 
-        await page.waitForTimeout(TIMEOUT_2_SECONDS);
-        await paddleFrame.cardNumberField.fill(CHECKOUT_CC_NUM, { timeout: TIMEOUT_2_SECONDS });
-        await paddleFrame.cardNameField.fill(CHECKOUT_CC_NAME, { timeout: TIMEOUT_2_SECONDS });
-        await paddleFrame.cardExpiryField.fill(CHECKOUT_CC_EXP, { timeout: TIMEOUT_2_SECONDS });
-        await paddleFrame.cardVerificationField.fill(CHECKOUT_CC_CVV, { timeout: TIMEOUT_2_SECONDS });
-
-        await Promise.all([page.waitForURL(ACCTS_CHECKOUT_SUCCESS_URL), await paddleFrame.finalCheckoutButton.click()]);
+        await page.waitForTimeout(TIMEOUT_5_SECONDS);
+        await paddleFrame.cardNumberField.fill(CHECKOUT_CC_NUM, { timeout: TIMEOUT_30_SECONDS });
+        await paddleFrame.cardNameField.fill(CHECKOUT_CC_NAME, { timeout: TIMEOUT_5_SECONDS });
+        await paddleFrame.cardExpiryField.fill(CHECKOUT_CC_EXP, { timeout: TIMEOUT_5_SECONDS });
+        await paddleFrame.cardVerificationField.fill(CHECKOUT_CC_CVV, { timeout: TIMEOUT_5_SECONDS });
 
+        await paddleFrame.finalCheckoutButton.click();
+        await page.waitForTimeout(TIMEOUT_5_SECONDS);
+        await page.waitForURL(ACCTS_CHECKOUT_SUCCESS_URL, { timeout: TIMEOUT_30_SECONDS });
         expect(page.url()).toBe(ACCTS_CHECKOUT_SUCCESS_URL);
```

Comment:
Does this need to await?

Reply:
No, I thought the same, but page.url() just returns a string with no promise.